### PR TITLE
Configuration options documented

### PR DESCRIPTION
**Pull request by alexeyvy**
https://oat-sa.atlassian.net/browse/TAO-8616

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,129 @@
...
+Add the extension to your TAO composer and to the autoloader:
+```bash
+composer require oat-sa/extension-tao-lti
+```
+
+Install the extension on the CLI from the project root:
+
+**Linux:**
+```bash
+sudo php tao/scripts/installExtension oat-sa/extension-tao-lti
```
Comment (bartlomiejmarszal) [inline README.md]: is it required to use `sudo` here?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,129 @@
...
+```
+
+**Windows:**
+```bash
+php tao\scripts\installExtension oat-sa/extension-tao-lti
+```
+
+As a system administrator you also install it through the TAO Extension Manager:
```
Comment (bartlomiejmarszal) [inline README.md]:
```suggestion
As a system administrator you can also install it through the TAO Extension Manager:
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,129 @@
...
+- Settings (the gears on the right hand side of the menu) -> Extension manager
```
Comment (bartlomiejmarszal) [inline README.md]: Can we use symbol/image here?

File: README.md (hunk 2 above)
Comment (bartlomiejmarszal) [inline README.md]: Why not use another heading as it is another way to install extension

File: README.md [hunk 4]
```diff
@@ -1,3 +1,129 @@
...
+- Select _taoLti_ on the right hand side, check the box and hit _install_
+
+## REST API
+[Lti REST API](https://openapi.taotesting.com/viewer/?url=https://raw.githubusercontent.com/oat-sa/extension-tao-lti/master/doc/rest.json)
+
+<!-- Uncomment and describe if applicable
+## LTI Endpoints
+-->
+
+## Extension Wiki
```
Comment (bartlomiejmarszal) [inline README.md]: I would keep this link below this document as annotation with link

File: README.md [hunk 5]
```diff
@@ -1,3 +1,129 @@
...
+You can find the [extension wiki here](https://github.com/oat-sa/extension-tao-lti/wiki).
+
+## Configuration options
+
+### auth.conf.php
+
+#### Configuration option `config`
+
+*Description:* specifies a single option as the `adapter` key of the array. This adapter is to be used to authenticate LTI requests and is retrieved in [FactoryLtiAuthAdapterService](taoLti/models/classes/FactoryLtiAuthAdapterService.php).
```
Comment (bartlomiejmarszal) [inline README.md]:
```suggestion
*Description:* this specifies a single option as the `adapter` key of the array. This adapter is to be used to authenticate LTI requests and is retrieved in [FactoryLtiAuthAdapterService](taoLti/models/classes/FactoryLtiAuthAdapterService.php).
```

File: README.md (hunk 1 above)
Comment (alexeyvy) [reply README.md]: I don't know, it's copied from here: https://github.com/oat-sa/readme-templates/tree/master/extension-core/extension-tao-lti

File: README.md [hunk 6]
```diff
@@ -1,3 +1,129 @@
...
+```bash
+php tao\scripts\installExtension oat-sa/extension-tao-lti
+```
+
+As a system administrator you also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoLti_ on the right hand side, check the box and hit _install_
+
+## REST API
+[Lti REST API](https://openapi.taotesting.com/viewer/?url=https://raw.githubusercontent.com/oat-sa/extension-tao-lti/master/doc/rest.json)
```
[the remark this answers is not in the document]
Comment (alexeyvy) [reply README.md]: I don't know, it's copied from here: https://github.com/oat-sa/readme-templates/tree/master/extension-core/extension-tao-lti

But I guess this is supposed to be a swagger viewer

File: README.md (hunk 3 above)
Comment (alexeyvy) [reply README.md]: what symbol?

File: README.md [hunk 7]
```diff
@@ -1,3 +1,129 @@
...
+*Possible values:* 
+* an instance of any class that implements the `oat\taoLti\models\classes\user\LtiUserFactoryInterface` interface
+
+#### Configuration option `transaction-safe` (only for `OntologyLtiUserService` implementation)
+
+*Description:* not used
+
+#### Configuration option `transaction-safe-retry` (only for `OntologyLtiUserService` implementation)
+
+*Description:* not used
```
[the remark this answers is not in the document]
Comment (alexeyvy) [reply README.md]: exists but is ignored in the code

File: README.md [hunk 8]
```diff
@@ -1,3 +1,129 @@
...
+#### Configuration option `factoryLtiUser`
+
+*Description:* factory for producing LTI users
+
+*Possible values:* 
+* an instance of any class that implements the `oat\taoLti\models\classes\user\LtiUserFactoryInterface` interface
+
+#### Configuration option `transaction-safe` (only for `OntologyLtiUserService` implementation)
+
+*Description:* not used
```
[the remark this answers is not in the document]
Comment (alexeyvy) [reply README.md]: exists but is ignored in the code

**Pull request merged**